src/code_gen.rs

```diff
@@ -98,7 +98,7 @@ pub struct RustUdf {
 #[derive(Serialize, PartialEq, Eq, Debug, Clone)]
 pub enum CppResult {
     Return { typ: CppType, id: String },
-    GroupBy { typ: CppType, id: String },
```

Comment:
It is bad design to have the two back ends in the same file. Ideally, they should be completely splt. I will work on this once I have more time.